### DISCUSSION TITLE
prowler aws inventory  check not pushing to S3 bucket

### DISCUSSION THREAD

**Discussion by Simon-Khoury** [prowler-cloud/prowler#2348] [Q&A] [answered]
im trying to run prowler container ( Inventory checks  -i )  with role assuming ( -R ) option and using (-D) to push to my own bucket and its not working ,  when i run it with security checks ( not using -i) its working

**Comment by MrCloudSec** [comment, marked as answer]
Hi @Simon-Khoury, this is now possible in the latest version 3.5.3.
Thanks for using Prowler!
